Keda.Cosmosdb.Scaler/src/Services/CosmosDBLeaseMetadata.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Keda.CosmosDB.Scaler.Services
+{
+    public class CosmosDBLeaseMetadata
+    {
+        public static string LeasesCosmosDBConnectionString { get { return "leasesCosmosDBConnectionString"; } }
+        public static string LeaseDatabaseName { get { return "leaseDatabaseName"; } }
+        public static string LeaseCollectionName { get { return "leaseCollectionName"; } }

```

Comment:
move these to constants

Reply:
Same comment here -> `Lease` & `CosmosDb` prefix are redundant thanks to the class name.

Thinks we ca remove this whole class in favor of the proposed constants by @pragnagopa